src/Microsoft.AspNetCore.Razor.Evolution/Intermediate/HtmlContentIRNodeExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.Intermediate
+{
+    internal static class HtmlContentIRNodeExtensions
+    {
+        private static StringBuilder _builder;
```

Comment:
Gonna revert this change and do this optimization as part of https://github.com/aspnet/Razor/issues/1202. I'll rename the issue title to something more appropriate.